cmd/convox/install.go

```diff
@@ -83,6 +83,12 @@ func init() {
 				Value: "",
 				Usage: "custom AMI for rack instances",
 			},
+			cli.StringFlag{
+				Name:   "build-instance",
```

Comment:
This is a style call, but this option is parallel to the `instance-type` option which raises some questions.

- [ ] Flag could be `build-instance-type`
- [ ] EnvVar could be RACK_BUILD_INSTANCE_TYPE
- [ ] Why an EnvVar for this at all?
- [ ] Why do some EnvVars use `RACK_` and others don't?